AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/FirebaseTestLabController.kt

```diff
@@ -49,16 +50,16 @@ class FirebaseTestLabController(
         val defaultModel = catalog.androidDeviceCatalog?.models?.first { model ->
             model.tags?.contains("default") == true
         } ?: error("Cannot find default model in test device catalog:  $catalog")
-        val defaultVersion = catalog.androidDeviceCatalog.versions?.first { version ->
-            version.tags?.contains("default") == true
-        } ?: error("Cannot find default version in test device catalog: $catalog")
+        val defaultModelVersion = defaultModel.supportedVersionIds?.maxByOrNull {
+            it.toIntOrNull() ?: -1
```

Comment:
What's the purpose of defaulting to -1 here, doesn't that just hide the error? I think we want:

```
val defaultModelVersion = defaultModel.supportedVersionIds
    ?.mapNotNull { it?.toIntOrNull() }
    ?.maxOfOrNull { it }
    ?: error("Cannot find supported version for $defaultModel in test device catalog: $catalog")
```

Reply:
you are mostly right. these ids are strings which I "assume" because of previews etc so not sure if all of it will be integers.  but i think we can assume it for now. I checked the latest catalog, all are ints.

Reply:
updated.